mkdocs.yml

```diff
@@ -4,33 +4,33 @@ nav:
     - Home: 'index.md'
     - Wiki: 'wiki/index.md'
     - Server Install and Configuration: 
-        - 'Prerequisities and Assumptions': 'wiki/00 - SERVER INSTALL AND CONFIGURATION/00 - Prerequisites And Assumptions/README.md'
-        - 'Database Configuration': 'wiki/00 - SERVER INSTALL AND CONFIGURATION/00 - Prerequisites And Assumptions/Database Configuration.md'
+        - 'Prerequisities and Assumptions': 'wiki/00_SERVER_INSTALL_AND_CONFIGURATION/00_Prerequisites_And_Assumptions/README.md'
+        - 'Database Configuration': 'wiki/00_SERVER_INSTALL_AND_CONFIGURATION/00_Prerequisites_And_Assumptions/Database_Configuration.md'
         - Installation:
-            - 'Ubuntu Install': 'wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md'
-            - 'CentOS Install': 'wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/CentOS/README.md'
+            - 'Ubuntu Install': 'wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md'
+            - 'CentOS Install': 'wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md'
         - Website Configuration: 
-            - 'Website Configuration': 'wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/README.md'
-            - 'Handling Large File Uploads': 'wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/Handling Large File Uploads.md'
-            - 'Enabling reCAPTCHA': 'wiki/99\ -\ Developers/reCAPTCHA.md docs/wiki/00\ -\ SERVER\ INSTALL\ AND\ CONFIGURATION/02\ -\ Website\ Configuration/reCAPTCHA.md'
+            - 'Website Configuration': 'wiki/00_SERVER_INSTALL_AND_CONFIGURATION/02_Website_Configuration/README.md'
+            - 'Handling Large File Uploads': 'wiki/00_SERVER_INSTALL_AND_CONFIGURATION/02_Website_Configuration/Handling_Large_File_Uploads.md'
+            - 'Enabling reCAPTCHA': 'wiki/99_Developers/reCAPTCHA.md docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/02_Website_Configuration/reCAPTCHA.md'
```

Comment:
- 'Enabling reCAPTCHA': 'docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/02_Website_Configuration/reCAPTCHA.md'

Reply:
@ .md files are lnked here, and the first one doesn't exist.

Reply:
@johnsaigle Should this be fixed? These 2 files were linked in the original version.